app/coffee/WebApiManager.coffee

```diff
@@ -24,7 +25,10 @@ module.exports = WebApiManager =
 			return callback(error) if error?
 			if 200 <= response.statusCode < 300
 				callback null, data?.project, data?.privilegeLevel
+			else if response.statusCode == 429
+				logger.log(project_id, user_id, "rate-limit hit when joining project")
+				callback(new CodedError("rate-limit hit when joining project", "TooManyRequests"))
 			else
+				logger.error {err, project_id, user_id}, "error accessing web api"
```

Comment:
`err` is undefined here. It will be defined in the next line, but you might want to log the actual `error` instead?

Reply:
Good spot, thanks.